webpack/assets/javascripts/react_app/routes/FiltersForm/FilterForm.scss

```diff
@@ -0,0 +1,6 @@
+.filter-form {
+  max-width: 880px;
```

Comment:
Until we use this fix https://github.com/patternfly/patternfly/pull/4782

Reply:
on what version would it be available? can you add a `TODO` with the issue link?